graphdatascience/algo/algo_proc_runner.py

```diff
@@ -25,7 +21,7 @@ def _run_procedure(self, G: Graph, config: Dict[str, Any]) -> DataFrame:
         return self._query_runner.run_query(query, params)
 
     def estimate(self, G: Graph, **config: Any) -> Series:
-        self._proc_name += "." + "estimate"
+        self._namespace += "." + "estimate"
```

Comment:
how does one call estimate ? gds.nodeClassification.train().estimate(config) ?
im confused because I'd guess train should take the config and run immediately

Reply:
For the pipelines you call `pipe.train_estimate(G, concurrency=4, metrics=[...],...)` to estimate